runtime/onert/frontend/circle_traininfo/circle_traininfo.fbs

```diff
@@ -0,0 +1,119 @@
+// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/91fc9550d914a3c62bf41e62e007e1a2f5f37b59/spec/circle_training.fbs
```

Comment:
Copy-past from circle-x repo, except this one line.
This line is for the future, to check the version between circle-x and this file.

Reply:
I follow the tflite-loader's overall structure

```
onert/frontend/tflite
	ㄴ include
		ㄴ tflite_loader.h
	ㄴ src
		ㄴ tflite_loader.cc
		ㄴ tflite_schema_generated.h
	ㄴ CMakeLists.txt
	ㄴ tflite_schema.fbs  
```

Reply:
As far as I know, training info is only used for circle. Can I know why training info is separated into a separate directory rather than the circle directory? Is this directory going to be changed to a directory for circle+?

Reply:
>  Is this directory going to be changed to a directory for circle+? 

No, I don't think so. As far as I know, there are NOT many things necessary for circle+.

> Can I know why training info is separated into a separate directory rather than the circle directory?

I tried to add new directory in the following context : 

First, it looks like each directory is responsible for parsing data based on each schema. 
(e.g circle directory is for circle schema, tflite directory is for tflite schema)
So I created a new traininfo directory for the circle-traininfo schema.  

Second, `circle_loader` is highly dependent on `base_loader`, but `traininfo_loader` isn't.
But `traininfo_loader` could parse buffer independently of `base_loader` and `circle_loader`.

ref) base_loader is necessary to build circle loader. 
https://github.com/Samsung/ONE/blob/ffe1c248974076f9f8d01bf8d5b1fce5f295a413/runtime/onert/frontend/circle/CMakeLists.txt#L13

Reply:
> As far as I know, training info is only used for circle. 

Yess. That's right.
I'm not opposite to move this into circle. 

Maybe we can move files to circle like this : 

```
➜ tree circle                                    

  |-- circle_traninfo.fbs (new!) 
 ├──      CMakeLists.txt  
 └──      include/ 
 │  └────      circle_loader.h
  |   ㄴ--  circle_traininfo_loader.h (new!) 
 └──      src/ 
 │  └────      circle_loader.cc  
      ㄴ--  circle_traininfo_loader.cc (new!) 
      ㄴ--  circle_traininfo_generated.h (new)!) 


```

Reply:
> I'm not opposite to move this into circle.

> Maybe we can move files to circle like this :

I didn't mean to move it into circle directory. I was just curious why training info was introduced as a separate schema. I think it would be better to place it in the current directory if the schema and the loader for training info are separated from circle.

Reply:
> No, I don't think so. As far as I know, there are NOT many things necessary for circle+.

I thought it would be better to manage the circle+ related schema in one file rather than introducing another schema each time for circle+. If there is no more schema to be added except for training info, this implementation looks good.